### ISSUE TITLE
Add tootctl equiv of refresh avatars for emojis

### ISSUE THREAD

**Issue by maximesrd** [tootsuite/mastodon#10610]
Following a server migration, after copying the system folder, it is possible to force the use of avatars already cached with the command: RAILS_ENV=production bin/tootctl account refresh --all

It would be interesting to have the same thing with emojis, for example: RAILS_ENV=production bin/tootctl emojis refresh --all

**Comment by ValiantNinja** [comment]
Do emojis change that much though? Wouldn't someone delete an emoji when updating it which would make more sense, and wouldn't that be updated anyway since you're deleting it and making a new one?

**Comment by maximesrd** [comment]
Yes, maybe, but if an emoji is stored only locally, it will be lost

**Comment by SuperSandro2000** [comment]
Bump as I want to solve this.

![image](https://user-images.githubusercontent.com/7258858/82317150-20c02f00-99ce-11ea-948e-8b2ae750fdcb.png)

**Comment by Brunty** [comment]
Restored a file backup without the cache and hit this problem, want to be able to reload the emojis.

It's not really much of a cache if you can't rebuild it easily, eh?

**Comment by keskival** [comment]
This is a related issue, also still open: https://github.com/mastodon/mastodon/issues/14681

**Comment by mkarg** [comment]
It would be nice if a committer could lay out the reasons why not fixing this bug after years? It is pretty annoying and there are open contributions that would fix it. Thanks.
